ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -108,6 +108,55 @@ tenants:
 +             always_redistribute_igmp: true
 ```
 
+### `vrf_id` will be used instead of `vrf_vni` to get the ip address of evpn underlay multicast group
+
+AVD versions below 5.0.0 uses `vrf_vni` in the algorithm to set the offset in order to get the ip address of evpn underlay multicast group.
+
+In AVD version 5.0.0 `vrf_vni` will be replaced with `vrf_id` to set the offset for ip address of evpn underlay multicast group when evpn_l3_multicast is enabled.
+
```

Comment:
```suggestion
AVD versions below 5.0.0 uses `vrf_vni` in the algorithm to set the offset in order to get the IP address of EVPN underlay multicast group.

In AVD version 5.0.0, `vrf_id` is used instead of `vrf_vni` to set the offset for IP address of EVPN underlay multicast group when `evpn_l3_multicast` is enabled.

```